CMakeLists.txt

```diff
@@ -25,8 +25,14 @@ endif()
 
 if(BUILD_ENTERPRISE)
     add_definitions(-DCOUCHBASE_ENTERPRISE)         # Tells CBL it's an EE build
```

Comment:
You won't need this anymore with the new file

Reply:
I was unsure how CMake builds the sub-targets like LiteCore and Fleece. I could try to remove this and see.